container-images/llama-stack/Containerfile

```diff
@@ -3,15 +3,17 @@ FROM registry.fedoraproject.org/fedora:42
 # hack that should be removed when the following PRs are merged
 # https://github.com/containers/ramalama-stack/pull/37
 # https://github.com/meta-llama/llama-stack/pull/2049
-RUN wget -x ~/.llama/providers.d/remote/inference/ramalama.yaml https://raw.githubusercontent.com/containers/ramalama-stack/refs/heads/main/providers.d/remote/inference/ramalama.yaml && \
-    wget -x ~/.llama/distributions/ramalama/ramalama-run.yaml https://raw.githubusercontent.com/containers/ramalama-stack/refs/heads/main/run.yaml
+RUN curl --create-dirs --output ~/.llama/providers.d/remote/inference/ramalama.yaml https://raw.githubusercontent.com/containers/ramalama-stack/refs/heads/main/providers.d/remote/inference/ramalama.yaml && \
```

Comment:
I was actually gonna ask could we just standardize on curl in the initial PR, but I went, meh, if wget works it's find